iris-machine-learning-tutorial/Makefile

```diff
@@ -0,0 +1,36 @@
+export AWS_ACCESS_KEY_ID ?= test
+export AWS_SECRET_ACCESS_KEY ?= test
+export AWS_DEFAULT_REGION = us-east-1
+
+usage:       ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+
+install:     ## Install dependencies
+	@which awslocal || pip install awscli-local
+
+start:
+	localstack start -d
+
+stop:
+	@echo
+	localstack stop
+
+ready:
+	@echo Waiting on the LocalStack container...
+	@localstack wait -t 30 && echo Localstack is ready to use! || (echo Gave up waiting on LocalStack, exiting. && exit 1)
+
+logs:
+	@localstack logs > logs.txt
+
+run:
+	awslocal ecr create-repository --repository-name iris-machine-learning-tutorial | jq -r '.repository.repositoryUri'
+	docker build -t localhost:4510/iris-machine-learning-tutorial .
```

Comment:
Better extract the port from `repositoryUri` here - in 99% of cases it will be `4510`, but it may also be a different port if users are not starting from a fresh instance (or are using a different external port range).